cmd/apps/certmanager_app.go

```diff
@@ -34,6 +34,7 @@ func MakeInstallCertManager() *cobra.Command {
 	certManager.RunE = func(command *cobra.Command, args []string) error {
 		wait, _ := command.Flags().GetBool("wait")
 		const certManagerVersion = "v0.12.0"
```

Comment:
Are these read from flags now?

Reply:
if it does it will error on any release after 0.14 as the CRDs moved

Reply:
its currently only pinned to 0.12, the pr from @meyskens to move to 0.15 allows version flag.